### ISSUE TITLE
Unable to transfer skills to a new character

### ISSUE THREAD

**Issue by muaddib76** [ChuckTheSheep/Skill-Recovery-Journal#5]
[console.txt](https://github.com/ChuckTheSheep/Skill-Recovery-Journal/files/6628190/console.txt)

As stated in the PZ workshop, the journal contains all the right skill XP but it doesn't transfer when read by a new character. Unsure if a separate mod is affecting it or some other issue. An error is presented when the journal is hovered over. I am running IWBUMS B41 Version 41.5

**Comment by ChuckTheSheep** [comment]
Does your player say there is nothing written? Are you able to take a screenshot of the tooltip on the journal.

**Comment by muaddib76** [comment]
Once I select the read option, my character begins the read action via flipping pages and once that action is complete nothing else occurs. Hovering over the bound journal does open the tooltip with all of my previous characters skills and their XP level. I will take a screen shot later and post here.

**Comment by ChuckTheSheep** [comment]
If you're seeing levels then you may have an older journal - I had to change the way it recorded. The tooltip should be "_skill_ (_n_ xp)". If you don't see this you may have to retranscribe the journal. Apologies.

**Comment by muaddib76** [comment]
Yeah, that’s actually what I do see. For instance: carpentry (24470 xp). I’ll take a screen shot of it later.

**Comment by ChuckTheSheep** [comment]
I notice in the log you have extraskills installed, I believe they change how books are read. I'll have to test ways around that.

**Comment by muaddib76** [comment]
Just to let you know, I turned off the extraskillz mod and that did the trick. All XP has been transferred as intended. Thanks

**Comment by ChuckTheSheep** [comment]
The latest update should work with extraskills. Let me know if there are issues.
